### DISCUSSION TITLE
When will version 2.0 be released?

### DISCUSSION THREAD

**Discussion by aswans** [digitalpetri/modbus#68] [General]
When will version 2.0 be released?

**Comment by kevinherron** [comment]
@aswans I will make the code and a `-SNAPSHOT` version available today.

**Reply by kevinherron** [reply]
Code is now available: https://github.com/digitalpetri/modbus/tree/dev/2.0

**Reply by kevinherron** [reply]
Merged into master.

**Reply by aswans** [reply]
Hope to have some more tutorials，I wish you more and more success in your open source projects 。 thank you!

**Reply by aswans** [reply]
Can your project run with Java 8?

**Reply by kevinherron** [reply]
Version 2.0 requires Java 17 or later.

**Reply by aswans** [reply]
Can Version 1.x  run with Java 8?

**Reply by kevinherron** [reply]
Yes
